Netw4ppl/app/src/main/java/netw4ppl/ines/utils/PersonDetailsRelationToAdapter.java

```diff
@@ -26,8 +27,8 @@ public class PersonDetailsRelationToAdapter  extends ArrayAdapter<Relation> {
      * A basic viewholder to put a person's name and a relation type
      */
     private static class ViewHolder {
-        TextView name_person;
-        TextView relation_type;
+        TextView bdv;
```

Comment:
This file is almost the same as PersonDetailsRelationFromAdapter, is that possible to merge them ?

Reply:
maybe with reflectivity ? #134

Reply:
guess so